docker-compose.dev.yml

```diff
@@ -13,7 +13,7 @@ services:
       - ../optinist-docker-volumes/logs/:/app/logs
       - ../optinist-docker-volumes/studio_data/:/app/studio_data
     ports:
-      - "127.0.0.1:8000:8000"
+      - "0.0.0.0:8000:8000"
```

Comment:
For security reasons, it is recommended that 0.0.0.0 not be opened by default in the development settings.
→ Please let me hear what kind of problem led you to the idea of opening 0.0.0.0.